packages/generator-react-native-vector-icons/src/app/templates/android/build.gradle

```diff
@@ -0,0 +1,69 @@
+def isNewArchitectureEnabled() {
```

Comment:
it'd be nice to have a comment at the top op this file that says something like

"this file was generated from packages/generator-react-native-vector-icons/src/app/templates/. If you're contributing to react-native-vector-icons, make the change there, otherwise it'll be lost."

that goes for all files here

Reply:
Have added this to files that allow it and have a note to make some improvements to CONTRIBUTING.md to explain how our build system works